plugin/compile-scss.js

```diff
@@ -257,14 +274,23 @@ function _getRealImportPathFromIncludes(importPath, getRealImportPathFn){
 
   _prepareNodeSassOptions();
 
-  let possibleFilePath, foundFile;
+  let possibleFilePath, foundFile, possibleFiles = [];
 
-  for (let includePath of _includePaths) {
-    possibleFilePath = path.join(includePath, importPath);
-    foundFile = getRealImportPathFn(possibleFilePath);
+  if (!importPath.match(/\.s?(a|c)ss$/)) {
+    for (const extension of expectedExtensions){
+      possibleFiles.push(`${importPath}.${extension}`);
+    }
+  } else {
+    possibleFiles.push(importPath);
+  }
 
-    if (foundFile) {
-      return foundFile;
+  for (let includePath of _includePaths) {
+    for (let possibleFile of possibleFiles) {
+      possibleFilePath = path.join(includePath, possibleFile);
+      foundFile = getRealImportPathFn(possibleFilePath);
+      if (foundFile) {
+        return foundFile;
+      }
```

Comment:
Same question here: Is this something required by `dart-sass` or is it an additional feature you like to include in `meteor-scss` package?

Reply:
Checked with my app, doesn't seem to be needed indeed.